### ISSUE TITLE
Example from documentation (digits) - score

### ISSUE THREAD

**Issue by mglowacki100** [automl/auto-sklearn#62]
I'm completly new to auto-sklearn. I've run example from documentation on digits dataset and print(automl.score...) gives me -0.0121288... instead of 0.98. I've noticed that "1 - 0.0121288"... is more or less 0.98, but I want to be sure that is not weird coincidence. 
If everything is correct, there should be note in documentation about this.

**Comment by mfeurer** [comment]
The score should not be negative, this looks like an error. What's the output of automl.show_models(), can you please also to calculate the accuracy with the scikit-learn metric?

**Comment by mfeurer** [comment]
Closing this because the issue stalled - if this issue persists with the latest version of auto-sklearn, please reopen this issue.
